game/atmosphere.rpy

```diff
@@ -49,142 +98,227 @@ init 0 python in jn_atmosphere:
         sunny = 3
         thunder = 4
         glitch = 5
+        snow = 6
 
     class JNWeather():
         def __init__(
             self,
             weather_type,
-            day_image,
-            night_image=None,
+            day_sky,
+            night_sky,
```

Comment:
Ideally keep the `image` or potentially use `_tag` to suffix since these are image tags. Makes it a little more clear to understand what they are at a glance since going between renpy/py code can be awkward, and knowing what's actually a string vs an image tag can get a little confusing